client/me/help/main.jsx

```diff
@@ -170,52 +157,61 @@ class Help extends React.PureComponent {
 					href="https://learn.wordpress.com"
 					target="__blank"
 				>
+					<Gridicon icon="list-ordered" size={ 36 } />
 					<div className="help__support-link-section">
-						<h2 className="help__support-link-title">
-							{ this.props.translate( 'Self-guided online tutorial' ) }
-						</h2>
+						<h2 className="help__support-link-title">{ this.props.translate( 'Guides' ) }</h2>
 						<p className="help__support-link-content">
 							{ this.props.translate(
 								'A step-by-step guide to getting familiar with the platform.'
 							) }
 						</p>
 					</div>
 				</CompactCard>
-				<CompactCard
-					className="help__support-link help__support-link-contact"
-					href="/help/contact/"
-				>
-					<div className="help__support-link-section">
-						<h2 className="help__support-link-title">{ this.props.translate( 'Get in touch' ) }</h2>
-						<p className="help__support-link-content">
-							{ this.props.translate(
-								"Can't find the answer? Drop us a line and we'll lend a hand."
-							) }
-						</p>
-					</div>
-					<Button className="help__support-link-button" primary>
-						{ this.props.translate( 'Contact Us' ) }
-					</Button>
-				</CompactCard>
 			</div>
-		);
-	};
+		</>
+	);
+
+	getContactUs = () => (
+		<>
+			<h2 className="help__section-title">{ this.props.translate( 'Contact Us' ) }</h2>
+			<CompactCard className="help__contact-us-card" href="/help/contact/">
+				<Gridicon icon="help" size={ 36 } />
+				<div className="help__contact-us-section">
+					<h3 className="help__contact-us-title">{ this.props.translate( 'Contact support' ) }</h3>
+					<p className="help__contact-us-content">
+						{ this.props.translate(
+							"Can't find the answer? Drop us a line and we'll lend a hand."
+						) }
+					</p>
+				</div>
+				<Button className="help__contact-us-button">
+					{ this.props.translate( 'Contact support' ) }
+				</Button>
+			</CompactCard>
+		</>
+	);
 
 	getCoursesTeaser = () => {
-		if ( ! this.props.showCoursesTeaser ) {
-			return null;
-		}
+		// @TODO: Find out if there are times we wouldn't want to show courses
```

Comment:
Is there any reasoning behind this removal? We previously were showing the courses only to business plan users:

https://github.com/Automattic/wp-calypso/blob/6b8d35840c658d943b88f9f19cd18dfb92d863ce/client/me/help/main.jsx#L274

The `isCoursesEnabled` prop comes from the value of the `help/courses` feature flag:

https://github.com/Automattic/wp-calypso/blob/6b8d35840c658d943b88f9f19cd18dfb92d863ce/client/me/help/controller.js#L45

But seems to be [enabled in all environments](https://github.com/search?q=%22help%2Fcourses%22+repo%3AAutomattic%2Fwp-calypso+extension%3Ajson&type=Code&ref=advsearch&l=&l=) at this moment, so I guess we could remove it a simplify the logic.

Reply:
> What's the reasoning behind this removal? We previously were showing the courses only to business plan users:

I didn't realize until I dug into this that webinars used to be the courses banner, and that it was a limited offering, so I laid everything out for four cards. I need to refactor the CSS so it works with either three cards or four depending on the plan type. :)

> But seems to be enabled in all environments at this moment, so I guess we could remove it and simplify the logic.

I looked through the commit history for this card and I don't see any explicit reasoning why it's behind a feature flag, so I think you're right. I can remove that flag and simplify the logic as part of this commit.